Polyhedron/demo/Polyhedron/Plugins/PMP/Selection_plugin.cpp

```diff
@@ -627,6 +743,7 @@ public Q_SLOTS:
     if (scene_ptr)
       connect(selection_item,SIGNAL(simplicesSelected(CGAL::Three::Scene_item*)), scene_ptr, SLOT(setSelectedItem(CGAL::Three::Scene_item*)));
     connect(selection_item,SIGNAL(isCurrentlySelected(Scene_polyhedron_item_k_ring_selection*)), this, SLOT(isCurrentlySelected(Scene_polyhedron_item_k_ring_selection*)));
+    on_LassoCheckBox_changed(ui_widget.lassoCheckBox->isChecked());
```

Comment:
This has a conflict with [that modification] from #1824.

[that modification]:  https://github.com/CGAL/cgal/pull/1824/files#diff-4fa131e055014434c9fc0405938df65e